### PR TITLE
Added 10km tolerance to geographic area validation for verbatim georeference

### PR DESCRIPTION
**Pull request by LocoDelAssembly**
This enables users to specify a geographic area other than the one that a verbatim georeference it is contained in provided it is not farther than 10km.

### CODE REVIEW

File: app/models/geographic_item.rb [hunk 1]
```diff
@@ -1067,7 +1086,7 @@ def intersects?(target_geo_object)
   # @param [geo_object]
   # @return [Boolean]
   def distance?(target_geo_object)
-    self.geo_object.distance?(target_geo_object)
```
Comment (mjy) [inline app/models/geographic_item.rb:1070]: Was this a uncaught error?

Comment (LocoDelAssembly) [reply app/models/geographic_item.rb:1070]: Yes, undefined method error. But I think this method is not actually being used (but I was hoping it was useful for me, it was not).

**Comment by LocoDelAssembly** [comment]
@mjy do you think this is good to go?

File: app/models/geographic_item.rb (hunk 1 above)
Comment (mjy) [reply app/models/geographic_item.rb:1070]: Go ahead and remove the method if it is not used.

File: app/models/georeference/verbatim_data.rb [hunk 2]
```diff
@@ -74,6 +74,32 @@ def dwc_georeference_attributes
     h
   end
 
+  # @return [Boolean]
+  #    true if geographic_item.geo_object is completely contained in collecting_event.geographic_area
+  # .default_geographic_item
+  def check_obj_within_distance_from_area(distance)
```
Comment (mjy) [inline app/models/georeference/verbatim_data.rb:80]: Probably nothing optimizable, but had me think a little about whether/how many times we have to hit the database here.  Would likely take more work than worthwhile at this point to tune things.

File: app/models/geographic_item.rb (hunk 1 above)
Comment (LocoDelAssembly) [reply app/models/geographic_item.rb:1070]: Other than `expect(geographic_item).to respond_to(:distance?)` doesn't seem to be used anywhere (and spec was only testing the method exists without actually executing it). This "go ahead" is for the entire PR or just removing this method?

**Comment by mjy** [comment]
@LocoDelAssembly couple very minor comments.  Form etc. is fine for me.

File: app/models/geographic_item.rb (hunk 1 above)
Comment (mjy) [reply app/models/geographic_item.rb:1070]: Remove it, and expectionation.

**Comment by mjy** [comment]
Looks good to merge, thanks much.

**Pull request merged**